### ISSUE TITLE
order table columns by date / alphabetically / etc.

### ISSUE THREAD

**Issue by jessedepagter** [TriliumNext/trilium#1586]
It took me a long, long time to find the perfect personal knowledge-base app and I am pretty sure Trilium is the one, great work!

Small question: In one of my notes I am using the "insert table" function in order to make a list of terms/and concepts with their definitions. The table has three columns: 'term/concept'; 'definition'; 'links for extra info'

I would like to be able to order the 'term/concept' column alphabetically. Would this be possible? I would also be interested in solutions outside of the table function.

**Comment by jessedepagter** [comment]
Hi, thanks for the quick answer :)
Ok it would be great if you find + integrate one at some point, but it's certainly not a major issue for me.

**Comment by zDEFz** [comment]
already have made a feature request at ckeditor sometime ago

**Comment by meichthys** [comment]
[Trilium has entered maintenance mode](https://github.com/zadam/trilium/issues/4620#issuecomment-2094495698). Future enhancements will be addressed in TrilumNext: https://github.com/TriliumNext/trilium/issues/4980

**Comment by meichthys** [comment]
Related functionality has been implemented in the table view of Collections.

**Comment by eliandoran** [comment]
Indeed, this functionality can be emulated via the Table collection.
We don't plan to alter the CKEditor table to add support for sorting since it's a lot of work.
